zmon_aws_agent/aws.py

```diff
@@ -748,6 +748,10 @@ def get_limits(region, acc, apps, elbs):
         'elb-max-count': 20,
         'elb-used-count': len(elbs),
     }
+    for k in previous:
+        if k.startswith('ec2-') or k.startswith('iam-') or k.startswith('rds-') or k.startswith('elb-'):
+            limits[k] = previous[k]
```

Comment:
I think we can use something like ``limits.update(e)`` and then set the computed value from ``apps`` and ``elbs``